hardware/email.sh

```diff
@@ -0,0 +1,12 @@
+# Example - {"ownerName":"balena-io-hardware", "repoName":"hardware-process-test", "branch":"dum-dum-pr", "installationId":15652884}
```

Comment:
Remind me, is this file used? Where? Subject seems a test one...

Reply:
yeah that file isn't used anymore as we've put that in the github release file. forgot i had that here. removed it.